examples/quickstart/jupyter-notebooks/docker-jupyter/docker-compose.yaml

```diff
@@ -0,0 +1,170 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+---
+version: "2.2"
+
+volumes:
+  metadata_data: {}
+  middle_var: {}
+  historical_var: {}
+  broker_var: {}
+  coordinator_var: {}
+  router_var: {}
+  druid_shared: {}
+
+
+services:
```

Comment:
Cool use of the `profiles` feature!

There is redundancy between this file and the previous one. Compose allows common services to be defined in a separate file, then imported. See the `dependencies.yaml` mentioned above, along with the corresponding `druid.yaml`.

Reply:
Correct. There was originally just the one file `docker-compose.yaml`, but we created the `docker-compose-local.yaml` version to give users an option to build the image locally from the repository. Otherwise users can pull the image from the third party Docker Hub.

Diff between the two files:
```
<     build:
<       context: ..
<       dockerfile: Dockerfile
---
>     image: imply/druid-notebook:latest
```